packages/webapp/src/webview/state/reducers.ts

```diff
@@ -261,6 +268,7 @@ function GuideFeedbackReducer(newState: AppState, action: GuideFeedback): AppSta
  */
 function setProductFiltersReducer(newState: AppState, action: SetProductFilters): AppState {
     newState.selectedProductFilters = action.payload;
+    newState.updatedGuidedAnswerTrees = [];
```

Comment:
This becomes obsolete when using `guidedAnswerTreeSearchResult.trees`
```suggestion
```

Reply:
Changed, thanks!